tests/unittests/test_format_selected_fields.py

```diff
@@ -1,14 +1,14 @@
 import singer
```

Comment:
Where are the other unit tests?

Reply:
These are the existing unittests which I have updated as per the new code. There is a separate PR #50 for adding unittests. I will merge that PR into this PR.

Reply:
Please merge it first. Will review it after.

Reply:
Merged the #50 into this PR.